MapboxDirections.xcodeproj/project.pbxproj

```diff
@@ -1134,9 +1263,11 @@
 			files = (
 				DAE9E0F41EB7DE2E001E8E8B /* RouteOptionsTests.swift in Sources */,
 				C5247D711E818A24004B6154 /* AnnotationTests.swift in Sources */,
+				C59666392048A20E00C45CE5 /* RoutableMatch.swift in Sources */,
```

Comment:
Rename this file to RoutableMatchTests.swift to match the class inside, RoutableMatchTests.